src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type MoviesArr = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<MoviesArr> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} key={movie.imdbId} />
```

Comment:
```suggestion
      <MovieCard movie={movie} key={movie.imdbId} />
```
you can also add movie to props instead of using spread operator